src/frontend/components/app/modal-info.tsx

```diff
@@ -0,0 +1,43 @@
+import { Modal } from '@adminjs/design-system'
+import React from 'react'
+import { useSelector } from 'react-redux'
+import { ModalButton } from '../../interfaces'
+import { ReduxState } from '../../store'
+import { useModal, useTranslation } from '../../hooks'
+
+export const ModalInfo = () => {
+  const rawModal = useSelector((state: ReduxState) => state.modal)
+  const { closeModal } = useModal()
+  const { translateButton, translateLabel, translateMessage } = useTranslation()
+  const modal = { ...rawModal }
+  const close = () => {
+    closeModal()
+  }
+  const confirm = () => {
+    closeModal()
+    modal.confirmAction()
+  }
+  const buttons: Array<ModalButton> = []
+  if (modal.type === 'confirm' && typeof modal.confirmAction === 'function') {
+    buttons.push({ label: translateButton('Cancel', modal.resourceId), variant: 'default', onClick: close })
```

Comment:
Add `Cancel`, `Confirm` etc to all locales. You can use https://www.deepl.com/pl/translator to translate.

`'Cancel'` -> `'cancel'`
Use lowercase camelcase for translation keys like in other translations

Reply:
done